spec/unit/postmark/api_client_spec.rb

```diff
@@ -606,7 +606,8 @@
   end
 
   describe "#server_info" do
-    let(:response) {{"Name" => "Testing",
+    let(:response) {
+      {"Name" => "Testing",
```

Comment:
@ibalosh this indentation looks off, was it on purpose? Same in the next chunk.

Reply:
hm, nah, wasn't on purpose, will fix it. In this project I do them manually, since auto corrections I normally use  on other projects (rubocop or Rubymine) might change it from current formatting ways.

I guess in future versions we should see to introduce some tooling/libraries for formatting code little bit, which still work in old Rubies.

Reply:
Yeah, totally. 
BTW: when you merge these PRs, consider using the "Squash and merge" option so that we will only have one commit per PR:
<img width="332" alt="Screen Shot 2021-07-21 at 08 44 57" src="https://user-images.githubusercontent.com/967638/126518645-562df7a6-99b9-449a-a0dd-af2bf4ca9508.png">

Reply:
nice! did not realize Github added that, thanks!